### PR TITLE
Working email verification system

### PR DESCRIPTION
**Pull request by ismailmohammad**
Will demo tomorrow at standup. Groundwork laid in for scheduling threshold alerts.
- Email verification System (there are likely missed edge cases to revisit but the basis of it is there)
- Isolated dependencies into requirements.txt for easy setup on new machines/eventually when containerized
- could use the send email functionality to send alerts etc. Right now it's very basic though, but we could eventually send nicer emails etc.

### CODE REVIEW

**Comment by dkaneria14** [comment]
@ismailmohammad hey can u leave out the package.lock JSON in the commit. 

Lets not push package.json

**Comment by dkaneria14** [comment]
@hanagkhan @AndreDevelopment 
check the FE and DB changes respectively

File: frontend/.nvmrc
Comment (hanagkhan) [inline frontend/.nvmrc:1]: What is this file for ?

File: frontend/src/components/AlertSignupModal.ts
Comment (hanagkhan) [inline frontend/src/components/AlertSignupModal.ts:1]: Where is this component being implemented?

File: frontend/src/App.js [hunk 1]
```diff
@@ -77,7 +78,25 @@ function App() {
...
+      percentChange: "+1.19",
+    },
+    AMZN: {
+      name: "Amazon.com, Inc",
+      ticker: "AMZN",
+      price: "127.74",
+      priceChange: "1.33",
+      percentChange: "-0.82",
+    },
+  };
```
Comment (ismailmohammad) [inline frontend/src/App.js:98]: This is only temporary. I can't seem to hit the yFinance API, results in a 404 but essentially when we have raw data to use, we can just use it to populate the fields.

Ideally imo, having TypeScript in the project would be nice as I've found that it helps with data modelling and enforcing it.

**Pull request merged**